types/v56/types.go

```diff
@@ -2066,10 +2066,10 @@ type NatOneToOneBasicRule struct {
 type NatOneToOneVMRule struct {
 	Xmlns string `xml:"xmlns,attr,omitempty"`
 	// Elements
-	MappingMode       string `xml:"MappingMode"`       // Mapping mode.
-	ExternalIPAddress string `xml:"ExternalIpAddress"` // External IP address to map.
-	VAppScopedVMID    string `xml:"VAppScopedVmId"`    // VAppScopedVmId of VM to which this rule applies.
-	VMNicID           int    `xml:"VmNicId"`           // VM NIC ID to which this rule applies.
+	MappingMode       string  `xml:"MappingMode"`       // Mapping mode.
+	ExternalIPAddress *string `xml:"ExternalIpAddress"` // External IP address to map.
```

Comment:
Why did we change from string to pointer?

Reply:
we will get error when we serialize empty value